docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,69 @@
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+Users of Netobserv CLI can pass command line arguments to enable features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+This document lists all the supported command line options and their possible values.
+
```

Comment:
== Basic commands

[cols="3a,8a",options="header"]
.Basic commands
|===
|Command| Description

| `flows`
| Capture flows information. For subcommands, see the "Flow capture subcommands" table. 

| `packets`
| Capture packets from a specific protocol or port pair, such as `netobserv packets --filter=tcp,80`. For more information about packet capture, see the "Packet capture subcommand" table.

| `cleanup`
| Remove the Network Observability CLI components.

| `version`
| Print the software version.

| `help`
| Show help.
|===

Reply:
Even though you can easily print this in the terminal, we should still add it for the sake of being thorough.